packages/manager-react-components/src/hooks/tags-list/useTagsFilter.tsx

```diff
@@ -0,0 +1,11 @@
+export const useTagsFilter = ({
+  tags,
+  displayInternalTags,
+}: {
+  tags: { [key: string]: string };
+  displayInternalTags: boolean;
+}) => {
+  return Object.keys(tags)
+    .filter((key) => displayInternalTags || key !== 'ovh')
+    .map((key) => `${key}:${tags[key]}`);
+};
```

Comment:
There is no added value to make it a hook: you don't use any hook or any context inside it
Just keep it as an util function